package.json

```diff
@@ -9,10 +9,11 @@
     "postinstall": "npm run build"
   },
   "dependencies": {
+    "dotenv": "^8.2.0",
```

Comment:
This doesn’t work?

https://github.com/UKHomeOfficeForms/hof-build/blame/master/README.md#L31